### PR TITLE
Load params sanity check

### PR DESCRIPTION
**Pull request by sevdarafatov**
Checking  if all the files and directory paths exists and are valid:

If parameter doesn`t exist in the configuration file giving  **The " " is not provided message**

If parameter is given but path is non existent giving a message arising from normalizePath

### CODE REVIEW

File: .gitignore
Comment (celinech) [inline .gitignore:1]: Do not modify the .gitignore.
We need to keep the docker-compose.yml to share how to use it.
You could make a copy in your requirements to keep it as you want, and let a standard copy in the repository.

File: load_parameters.R
Comment (celinech) [inline load_parameters.R:1]: It is good to check if the variable exists :+1: 
Can you add the existence of the path/file also ?
For instance, one could set the `PATH_ATLAS = "vhiameoivnaq"`
The current file will assign the variable with the value, but the directory doesn't exist :(

Doing both in the same branch and in the same PR makes sense. What do you think ?

**Review by celinech — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (sevdarafatov) [reply .gitignore:1]: Okay!

File: 08_DEG_analysis.Rmd
[the remark this answers is not in the document]
Comment (celinech) [reply 08_DEG_analysis.Rmd:1]: yes, right :+1:

File: 00_pipeline_launcher.R
Comment (sevdarafatov) [inline 00_pipeline_launcher.R:1]: I think this part should be commented out? ""opt$options$input_list <- "cardioWT,cardioKO"

**Pull request merged**